contracts/WitnetRequestsBoardProxy.sol

```diff
@@ -0,0 +1,122 @@
+pragma solidity ^0.5.0;
+
+import "./WitnetRequestsBoardInterface.sol";
+
+
+/**
+ * @title Block Relay Proxy
+ * @notice Contract to act as a proxy between the Witnet Bridge Interface and the block relay
+ * @dev More information can be found here
+ * DISCLAIMER: this is a work in progress, meaning the contract could be voulnerable to attacks
+ * @author Witnet Foundation
+ */
+contract WitnetRequestsBoardProxy {
+
+  address public witnetRequestsBoardAddress;
+  WitnetRequestsBoardInterface witnetRequestsBoardInstance;
+  // Last id of the previous WRB controller
+  uint256 lastId;
+  // Map the lastIds with the corresponding WRB controllers
+  mapping(uint256 => address)  idWrb;
+  // Array of the lastIds of the controllers
+  uint256[] lastIdsControllers;
+  // OffSet for having unique ids in the Proxy
+  uint256 offSet;
+
+  modifier notIdentical(address _newAddress) {
+    require(_newAddress != witnetRequestsBoardAddress, "The provided Witnet Requests Board instance address is already in use");
+    _;
+  }
+
+  constructor(address _witnetRequestsBoardAddress) public {
+    witnetRequestsBoardAddress = _witnetRequestsBoardAddress;
+    witnetRequestsBoardInstance = WitnetRequestsBoardInterface(_witnetRequestsBoardAddress);
+    // Set the first possition of the lastIdsControllers to zero
+    lastIdsControllers = [0];
+  }
```

Comment:
Push the _witnetRequestsBoardAddress to the mapping/strcuture too as initialization